### PR TITLE
Update emoji dataset with skin tone variants

### PR DESCRIPTION
**Pull request by mrsdizzie**
Sorry last change I didn't have time to stick into previous PR:

Since the format of emoji that support skin tone modifiers is predictable we can add different variants into our dataset when generating it so that we can match and properly style most skin tone variants of emoji. No real code change here other than what generates the dataset and the data itself.

### CODE REVIEW

File: build/generate-emoji.go [hunk 1]
```diff
@@ -102,18 +106,20 @@ func generate() ([]byte, error) {
 		return nil, err
 	}
 
-	var re = regexp.MustCompile(`keycap|registered|copyright`)
-	tmp := data[:0]
+	var skinTones = make(map[string]string)
 
-	// filter out emoji that require greater than max unicode version
+	skinTones["🏻"] = "Light Skin Tone"
```
Comment (zeripath) [inline build/generate-emoji.go]: Probably best to use \uxxxx notation for these

Comment (mrsdizzie) [reply build/generate-emoji.go]: done

**Review by silverwind — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
Ping lg-tm

**Comment by mrsdizzie** [comment]
Should be backport 1.12 I think

**Pull request merged**